testbox

```diff
@@ -111,7 +111,11 @@ unit)
 codegen)
   codegen
 ;;
-e2e) operator-sdk test local ./e2e --debug --verbose --image quay.io/tmaxanc/kubevirt-image-service:canary;;
+e2e) 
+  kubectl create ns kis
```

Comment:
we have https://github.com/tmax-cloud/kubevirt-image-service/blob/master/deploy/namespace.yaml
we can use this.